packages/vscode-extension/src/extension/main-browser.ts

```diff
@@ -80,4 +80,8 @@ function registerFileSystemProvider(client: LanguageClient): void {
       return result.map((r) => r[0]);
     });
   });
+  client.onRequest(Messages.WriteFile, (uriString, value) => {
+    const uri = vscode.Uri.parse(uriString);
+    vscode.workspace.fs.writeFile(uri, value);
```

Comment:
Minor: Await this function call
```suggestion
    await vscode.workspace.fs.writeFile(uri, value);
```